pkgs/development/coq-modules/coq-elpi/default.nix

```diff
@@ -6,20 +6,21 @@ with builtins; with lib; let
     { case = "8.12"; out = { version = "1.12.0"; };}
     { case = "8.13"; out = { version = "1.13.7"; };}
     { case = "8.14"; out = { version = "1.13.7"; };}
-    { case = "8.15"; out = { version = "1.13.7"; };}
+    { case = "8.15"; out = { version = "1.14.1"; };}
   ] {});
 in mkCoqDerivation {
   pname = "elpi";
   repo  = "coq-elpi";
   owner = "LPCIC";
   inherit version;
   defaultVersion = lib.switch coq.coq-version [
-    { case = "8.15"; out = "1.12.1"; }
+    { case = "8.15"; out = "1.13.0"; }
     { case = "8.14"; out = "1.11.2"; }
     { case = "8.13"; out = "1.11.1"; }
     { case = "8.12"; out = "1.8.3_8.12"; }
     { case = "8.11"; out = "1.6.3_8.11"; }
   ] null;
+  release."1.13.0".sha256     = "sha256-BZPVL8ymjrE9kVGyf6bpc+GA2spS5JBpkUtZi04nPis=";
```

Comment:
```suggestion
  release."1.13.0".sha256     = "sha256-9vATtP8gds9m/KhyaODp2YfJZH1S52WfEIIvaWk7+sg=";
```

Reply:
Hum, I did push before seeing this suggestion, using the hash printed by CI. Hope it is still ok.

Reply:
You had put the same hash for the two tarballs (elpi and coq-elpi).